pennylane_lightning/lightning_gpu/lightning_gpu.py

```diff
@@ -86,7 +86,31 @@
         "PauliZ",
         "MultiRZ",
         "GlobalPhase",
+        "C(PauliX)",
+        "C(PauliY)",
+        "C(PauliZ)",
+        "C(Hadamard)",
+        "C(S)",
+        "C(T)",
+        "C(PhaseShift)",
+        "C(RX)",
+        "C(RY)",
+        "C(RZ)",
+        "C(Rot)",
+        "C(SWAP)",
+        "C(IsingXX)",
+        "C(IsingXY)",
+        "C(IsingYY)",
+        "C(IsingZZ)",
+        "C(SingleExcitation)",
+        "C(SingleExcitationMinus)",
+        "C(SingleExcitationPlus)",
+        "C(DoubleExcitation)",
+        "C(DoubleExcitationMinus)",
+        "C(DoubleExcitationPlus)",
+        "C(MultiRZ)",
         "C(GlobalPhase)",
+        "C(QubitUnitary)",
```

Comment:
I think the PL name is different in this case: 
```suggestion
        "ControlledQubitUnitary",
```